### PR TITLE
MONK: cleaned and update T24 default flask logic

### PR DESCRIPTION
**Pull request by kendrick-walls-work**
cleaned and update T24 default flask logic all 111+
rational: mistweaver monks should use same flask, no need for special flask for 119-120

ℹ️ related PR SimulationcCraft/simc#4949
- This PR set also obsoletes also obsoletes PRs SimulationcCraft/simc#4852 and SimulationcCraft/simc#4948 in a clean way

### CODE REVIEW

**Comment by kendrick-walls-work** [comment]
:x: blocked by PR SimulationCraft/simc#4949 and waiting on SimulationCraft/simc#4949 being merge

**Comment by kendrick-walls-work** [comment]
👍 Ready for merge

File: engine/class_modules/sc_death_knight.cpp [hunk 1]
```diff
@@ -7535,7 +7535,7 @@ std::string death_knight_t::default_food() const
                             "disabled";
 
   std::string blood_food =  ( true_level > 110 ) ? "mechdowels_big_mech" :
-	                          ( true_level > 100 ) ? "lavish_suramar_feast" :
+	                    ( true_level > 100 ) ? "lavish_suramar_feast" :
```
Comment (kendrick-walls-work) [inline engine/class_modules/sc_death_knight.cpp:7538]: Please ignore this trivial change to formatting, this has no impact on function.

**Comment by seanpeters86** [comment]
closing due to inactivity, feel free to reopen if still needed

**Pull request closed without merging**